src/pages/login/login.js

```diff
@@ -50,6 +54,22 @@ const Login = () => {
       .required(' ')
   });
 
+  useEffect(() => {
```

Comment:
If we need it to init only once you should write it with empty dependencies array: **useEffect(()=>{...},**[]**)**

Reply:
Fixed